content/docs/for-platform-operators/tutorials/lifecycle/_index.md

```diff
@@ -5,4 +5,177 @@ expand=true
+
+### Build the lifecycle
+
+As a starting step, you need to build the `lifecycle` in order to use its phases. This could be done by navigating to the `lifecycle` directory and executing one of the following commands, depending on your system architecture.
+
+* `make build` for `AMD64` architectures
+* `make build-darwin-arm64` for `ARM64` architectures
+
+### Set environment variables
+
+In order to execute the various `lifecycle phases` correctly, you first need to set the values of few important environment variables by running the following commands in the terminal:
+
+```text
+
+export CNB_USER_ID=1000 CNB_GROUP_ID=1000 CNB_PLATFORM_API=0.14
+export CNB_SAMPLES_PATH="/<your-path>/samples"
+export CNB_LIFECYCLE_PATH="/<your-path/lifecycle/out/<your-arch>/lifecycle"`
+
+```
+
+Where
+
+* `CNB_USER_ID` and `CNB_GROUP_ID` are arbitrary values that need to be consistent, which both have a default value of `1000`.
+* `CNB_PLATFORM_API` or the `Platform API` version, varies depending on the use case. This tutorial uses `v0.14`, which is the latest [Platform API][Platform API] version.
+* `CNB_SAMPLES_PATH` represents the path of our local copy of the `samples` directory.
+* `CNB_LIFECYCLE_PATH` represents the path of our local compiled `lifecycle` directory.
+
+### Examine lifecycle phases
+
+A single app image build consists of the following phases:
+
+1. [Analysis](#analyze)
+2. [Detection](#detect)
+3. [Cache Restoration](#restore)
+4. [Build](#build)
+5. [Export](#export)
+
+> Note that a `platform` executes the phases above either by invoking phase-specific lifecycle binaries in order or by executing `/cnb/lifecycle/creator`.
+
+Let's expand each `lifecycle` phase to explain how the `lifecycle` orchestrates buildpacks:
+
+#### Analyze
+
+The `analyze` phase runs before the `detect` phase in order to validate registry access for all images used during the `build` as early as possible. In this way it provides faster failures for end users.
+
+Prior to executing `/cnb/lifecycle/analyzer`, you need to create two directories in the `root` directory as follows:
+
+```text
+
+mkdir -p apps/bash-script
+mkdir -p layers
+
+```
+
+* `apps` directory that contains a `bash-script` directory
+* `layers` directory that contains subdirectories representing each layer created by the Buildpack in the final image or build cache.
+
+Next,  you need to copy the `bash-script` samples into our `apps/bash-script` directory, which will host our app's source code.
+
+```text
+
+cp -r "$CNB_SAMPLES_PATH/apps/bash-script/" ./apps/bash-script
+
+```
+
+Now, you can invoke the `analyzer` for `AMD64` and `ARM64` architectures respectively
+
+```text
+
+$CNB_LIFECYCLE_PATH/analyzer -log-level debug -daemon -layers="./layers" -run-image cnbs/sample-stack-run:bionic apps/bash-script
+
+```
+
+```text
+
+$CNB_LIFECYCLE_PATH/analyzer -log-level debug -daemon -layers="./layers" -run-image arm64v8/ubuntu:latest apps/bash-script
```

Comment:
```suggestion
${CNB_LIFECYCLE_PATH}/analyzer -log-level debug -daemon -layers="./layers" -run-image arm64v8/ubuntu:latest apps/bash-script
```

Note to @AidanDelaney, double check this run image.

Reply:
updated